packages/client/src/forms/user/query/queries.ts

```diff
@@ -11,35 +11,25 @@
 import { gql } from '@apollo/client'
 import { client } from '@client/utils/apolloClient'
 
-export const getSystemRolesQuery = gql`
-  query getSystemRoles($value: ComparisonInput) {
```

Comment:
Not sure why we had these filtering and search parameters here in the first place